docs/sql-programming-guide.md

```diff
@@ -1766,12 +1806,6 @@ Spark SQL supports the vast majority of Hive features, such as:
 Below is a list of Hive features that we don't support yet. Most of these features are rarely used
 in Hive deployments.
 
-**Major Hive Features**
-
-* Tables with buckets: bucket is the hash partitioning within a Hive table partition. Spark SQL
-  doesn't support buckets yet.
```

Comment:
We do support buckets, but it is slightly different from Hive. See the ongoing PR:  https://github.com/apache/spark/pull/17644

Could you document the difference too? Thanks!